src/main/java/dev/jbang/catalog/CatalogRef.java

```diff
@@ -52,9 +52,10 @@ static CatalogRef get(String catalogName) {
 		if (catalog != null) {
 			catalogRef = catalog.catalogs.get(catalogName);
 		}
-		if (catalogRef == null && Util.isValidPath(catalogName)) {
+		if (catalogRef == null && (!catalogName.startsWith("/") || !catalogName.equals(""))
```

Comment:
this is causing the test failure...we are for some reason resolving local paths first to support `somealias@/var/path/locally/ondisk` ?

@quintesse what usecase is that for ?

Reply:
https://github.com/jbangdev/jbang/pull/1363 is what introduced `bar@/some/file/on/disk`

Reply:
I've updated the code to make it so paths that exist on disk will be honorored.

But I'm not too happy about it.

its only needed for when you use absolute paths...relative paths would be different.

I could also let it run through first and then if fails lookup in jbanghub as a fallback - to avoid scanning multiple times.

Reply:
could accept file:// instead of raw path ?

Because as it is now a local dir will actually override an implicit catalog. 

Jbang myapp@jbanghub and you have jbanghub locally will override jbanghub catalog.

Reply:
> could accept file:// instead of raw path ?

We could do that, yes. It does go somewhat against the fact that generally speaking a catalog reference is also a `name@resourceref` thing, so almost anything you should be able to pass to for example a `//FILE` tag was possible to use in a catalog reference. We'd now be limiting that. Now, I don't think anyone was actually using this, so we most likely won't be breaking anything, but just be aware that we _will_ be making a breaking change.

Reply:
Ok, I just checked the URI spec, it seems that `file:` doesn't allow relative paths, so in that respect using `file:` is not a good replacement. And that _does_ seem somewhat wrong. You'd be forced to always use absolute paths.

Reply:
hmm - I thought `file:/./mydir` was allowed but yeah does not seem to be "official" syntax.

URI.create("file:./pom.xml").getSchemeSpecficiPart() does let you fetch it though... but yeah not great.